solutions/telemetry/README.md

```diff
@@ -0,0 +1,10 @@
+# Telemetry App Examples
```

Comment:
Why not having the `solutions` folder inside each app? That way when you clone & move to the app directory (or even copy the app).. you end having the app + the solution in a known and very close place.

If for example say students to check the solution file, the path to the file can be relative.

Reply:
Nice idea, I like it.
We have to be aware that if we do it that way... then we can't reuse apps. Every single ge/lab will have its own folder, which is not necessarily a bad thing